src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -260,13 +260,14 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
 
                             const store = this.resultsViewPageStore;
 
-                            if (store.rnaSeqMolecularData.isComplete && store.studyIdToStudy.isComplete
-                                && store.mutations.isComplete && store.genes.isComplete && store.coverageInformation.isComplete) {
+                            if (store.studyIdToStudy.isComplete
+                                && store.putativeDriverAnnotatedMutations.isComplete && store.genes.isComplete && store.coverageInformation.isComplete) {
```

Comment:
putative driver contains rnaseq and mutations data?

Reply:
what?

Reply:
oh, no I just rewrote it to use numericGeneMolecularDataCache as a data source